buffer/src/main/java/io/netty/buffer/ByteBufUtil.java

```diff
@@ -1027,7 +1027,7 @@ private static int safeArrayWriteUtf8(byte[] buffer, int writerIndex, CharSequen
         return writerIndex - oldWriterIndex;
     }
 
-    // unsafe Fast-Path implementation
+        // unsafe Fast-Path implementation - for buffer + (long) offset
     private static int unsafeWriteUtf8(byte[] buffer, long memoryOffset, int writerIndex,
```

Comment:
since this is a private method and we are porting this to use MemoryAddress, the old function can be removed?

Reply:
I'm a little rusty on this as it's quite a while since I've looked at it - but isn't the `byte[] array, long offsetIntoBuffer` approach slightly more performant than accessing the memory via a `MemoryAddress`? And if so, it's probably best to keep both implementations?